_search-plugins/ubi/ubi-dashboard-tutorial.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: UBI Dashboard Tutorial
+parent: User behavior insights
+has_children: false
+nav_order: 7
+---
+
+
+# Build an analytic dashboard for UBI
+
+Whether you've been collecting user events and queries for a while, or [you uploaded some sample events](https://github.com/o19s/chorus-OpenSearch-edition/blob/main/katas/003_import_preexisting_event_data.md), now you're ready to visualize them in an OpenSearch Dashboards dashboard using User Behavior Insights (UBI).
+
+To view the resulting dashboard without following the tutorial, follow these steps:
+1. Download and save the [sample UBI dashboard]({{site.url}}{{site.baseurl}}/assets/examples/ubi-dashboard.ndjson).
+1. On the top menu, go to **Management > Dashboard Management**.
+1. From the **Dashboards** panel, choose **Saved objects**.
+1. In the upper right corner, select **Import**.
+1. In the **Select file** panel, choose **Import**.
+1. Select the UBI dashboard file that you downloaded and select the **Import** button.
+
+## 1. Start OpenSearch Dashboards
+
+Depending on your configuration: `http://{server}:5601/app/home#/`
+![Dashboard Home]({{site.url}}{{site.baseurl}}/images/ubi/home.png "Dashboards")
+{: .note} 
+
+## 2. Create an index pattern
+
+Within OpenSearch Management navigate to **Dashboards Management > Index patterns**.
+
+`http://{server}:5601/app/management/OpenSearch-dashboards/indexPatterns`
+
+It will look something like the following.
```

Comment:
Is this here by mistake?

Reply:
ah, the image was suggested to be removed